### PR TITLE
Fix opentracing env usage

### PR DESCRIPTION
**Pull request by denis-tingajkin**
## Motivation and Context
1) Adds possible to disable opentracing for containers which uses opentracing.
2) Unifies the check is opentracing enabled

## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [ ] Added integration testing to cover
- [ ] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.

### CODE REVIEW

File: sdk/common/configuration.go [hunk 1]
```diff
@@ -80,7 +78,7 @@ func (configuration *NSConfiguration) CompleteNSConfiguration() {
 		configuration.OutgoingNscLabels = getEnv(outgoingNscLabelsEnv, "Outgoing labels", false)
 	}
 
-	configuration.TracerEnabled, _ = strconv.ParseBool(getEnv(tracerEnabled, "Tracer enabled", false))
+	configuration.TracerEnabled = tools.IsOpentracingEnabled()
```
Comment (haiodo) [inline sdk/common/configuration.go]: @denis-tingajkin I think we do not need this property into configuration?

Comment (denis-tingajkin) [reply sdk/common/configuration.go]: thanks, fixed

File: controlplane/cmd/nsmd/main.go [hunk 2]
```diff
@@ -35,9 +35,12 @@ func main() {
 	// Capture signals to cleanup before exiting
 	c := tools.NewOSSignalChannel()
 
-	tracer, closer := tools.InitJaeger("nsmd")
-	opentracing.SetGlobalTracer(tracer)
-	defer closer.Close()
+	if tools.IsOpentracingEnabled() {
```
Comment (edwarnicke) [inline controlplane/cmd/nsmd/main.go]: Nice touch :)

**Pull request merged**